client/src/components/organisms/WalletSelector.module.scss

```diff
@@ -24,7 +24,7 @@
     align-items: flex-start;
     text-align: left;
     cursor: pointer;
-    transition: border .2s ease-out;
+    transition: border 0.2s ease-out;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)